graph2rdf/res/embeddings.csv

```diff
@@ -0,0 +1,14 @@
+iri,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45
```

Comment:
what is this file used for? can it be deleted?

Reply:
This can be deleted.